.circleci/config.yml

```diff
@@ -30,6 +30,7 @@ jobs:
             nvm install $node_version
             nvm use $node_version
             npm install
+            npm install github:Displayr/rhtmlBuildUtils#revert-dependabot
```

Comment:
Even though this version is listed in package.json for some reason it doesn't seem to be used so this extra install step is required to use our modified version of rhtmlBuildUtils